src/lib.rs

```diff
@@ -49,10 +49,21 @@ pub fn parse_line<S: AsRef<str>>(line: S) -> Option<TestResult> {
     if line.starts_with("result") || line.contains("(line ") {
         return None;
     }
-    let line = MODULE_PREFIX.replace(line, "");
+
+    let (module, line) = match MODULE_PREFIX.captures(line) {
```

Comment:
This works, but it feels a bit fiddly, doesn't it? I was wondering if we could do something with the capture groups instead: you know, capture the module name separately to the test name. Then we wouldn't have to disassemble it here, and we could use `match` captures instead. Something like this:

```rust
Some(&[module, test]) => ...
Some(&[test]) => ...
```

Is that workable? If not, we'll stick with what we have for now.